<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1352</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login Page</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(194, 154, 191);
background-color: rgb(253, 245, 223);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="column" stretch="1,5,0">
     <property name="spacing">
      <number>40</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>40</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item alignment="Qt::AlignHCenter|Qt::AlignTop">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Login Page</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QWidget" name="loginsWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>1250</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="loginsLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>380</width>
              <height>344</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>600</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget{background-color: rgb(0, 85, 127); border: 2px solid white; border-radius: 20}

</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,2,2,2">
               <property name="sizeConstraint">
                <enum>QLayout::SetMinimumSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="lblStudent_4">
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Student Login</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignHCenter|Qt::AlignTop</set>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <property name="leftMargin">
                  <number>12</number>
                 </property>
                 <property name="rightMargin">
                  <number>12</number>
                 </property>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLabel" name="label_8">
                   <property name="styleSheet">
                    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
                   </property>
                   <property name="text">
                    <string>Enter Username: </string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLineEdit" name="inputUsernameStudent">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="maxLength">
                    <number>20</number>
                   </property>
                   <property name="placeholderText">
                    <string>exampleStudent</string>
                   </property>
                   <property name="clearButtonEnabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <property name="leftMargin">
                  <number>12</number>
                 </property>
                 <property name="rightMargin">
                  <number>12</number>
                 </property>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLabel" name="label_9">
                   <property name="styleSheet">
                    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="text">
                    <string>Enter password: </string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLineEdit" name="inputPasswordStudent">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="inputMask">
                    <string/>
                   </property>
                   <property name="maxLength">
                    <number>32767</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="btnStudentLogin">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>400</width>
                   <height>80</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Student Login</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>380</width>
              <height>344</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>600</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget{background-color: rgb(0, 85, 127); border: 2px solid white; border-radius: 20;}

</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,2,2,2">
               <property name="sizeConstraint">
                <enum>QLayout::SetMinimumSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="lblStudent_6">
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Company Login</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignHCenter|Qt::AlignTop</set>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <property name="leftMargin">
                  <number>12</number>
                 </property>
                 <property name="rightMargin">
                  <number>12</number>
                 </property>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLabel" name="label_12">
                   <property name="styleSheet">
                    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
                   </property>
                   <property name="text">
                    <string>Enter Username: </string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLineEdit" name="inputUsernameCompany">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="maxLength">
                    <number>20</number>
                   </property>
                   <property name="placeholderText">
                    <string>exampleCompany</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <property name="leftMargin">
                  <number>12</number>
                 </property>
                 <property name="rightMargin">
                  <number>12</number>
                 </property>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLabel" name="label_13">
                   <property name="styleSheet">
                    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="text">
                    <string>Enter password: </string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLineEdit" name="inputPasswordCompany">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="maxLength">
                    <number>20</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="btnCompanyLogin">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>400</width>
                   <height>80</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Company Login</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="minimumSize">
             <size>
              <width>380</width>
              <height>344</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>600</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget{background-color: rgb(0, 85, 127); border: 2px solid white; border-radius: 20}
</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,2,2,2">
               <property name="sizeConstraint">
                <enum>QLayout::SetMinimumSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="lblStudent_5">
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Instructor Login</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignHCenter|Qt::AlignTop</set>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <property name="leftMargin">
                  <number>12</number>
                 </property>
                 <property name="rightMargin">
                  <number>12</number>
                 </property>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLabel" name="label_10">
                   <property name="styleSheet">
                    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
                   </property>
                   <property name="text">
                    <string>Enter Username: </string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLineEdit" name="inputUsernameInstructor">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="maxLength">
                    <number>20</number>
                   </property>
                   <property name="placeholderText">
                    <string>exampleInstructor</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_11">
                 <property name="leftMargin">
                  <number>12</number>
                 </property>
                 <property name="rightMargin">
                  <number>12</number>
                 </property>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLabel" name="label_11">
                   <property name="styleSheet">
                    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="text">
                    <string>Enter password: </string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLineEdit" name="inputPasswordInstructor">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="maxLength">
                    <number>20</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="btnInstructorLogin">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>400</width>
                   <height>80</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Instructor Login</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnGoBack">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(125, 188, 0);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icon_back.jpg</normaloff>icon_back.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
